src/charts/heatmap/heatmap.directive.js

```diff
@@ -246,14 +261,34 @@ angular.module('patternfly.charts').directive('pfHeatmap', function ($compile, $
       var redraw = function () {
         var data = scope.data;
         var color = d3.scale.threshold().domain(scope.thresholds).range(scope.heatmapColorPattern);
+        var rangeTooltip = d3.scale.threshold().domain(scope.thresholds).range(scope.rangeTooltips);
         var blocks;
         var fillSize = blockSize - scope.padding;
         var highlightBlock = function (block, active) {
           block.style('fill-opacity', active ? 1 : 0.4);
         };
+        var highlightBlockColor = function (block, fillColor) {
+          // Get fill color from given block
+          var blockColor = color(block.map(function (d) {
+            return d[0].__data__.value;
+          }));
+          // If given color matches, apply highlight
+          if (blockColor === fillColor) {
+            block.style('fill-opacity', 1);
+          }
+        };
+
+        // Highlight range on hover when default > fill size or when rangeOnHover is true
+        var highlightRange = ((scope.rangeOnHover === undefined && fillSize < 16) || scope.rangeOnHover) ? true : false;
```

Comment:
Currently, the feature can be forced on using rangeOnHover=true. This overrides the default value of 16. Similarly, it can be completely disabled using rangeOnHover=false.

I actually had both max-block-size="15" and max-block-hover-size="16" in my initial example. However, it wouldn't be enabled unless max-block-size < max-block-hover-size. It didn't make a lot of sense/confusing to have both. I suppose it might be less confusing if max-block-size wasn't used.

Should I can add this back in?